editor.planx.uk/src/pages/FlowEditor/components/Subscription/components/Discount.tsx

```diff
@@ -0,0 +1,88 @@
+import Box from "@mui/material/Box";
+import Typography from "@mui/material/Typography";
+import { Gauge, gaugeClasses } from "@mui/x-charts/Gauge";
+import { formattedPriceWithCurrencySymbol } from "@planx/components/Pay/model";
+import { DEFAULT_SERVICE_CHARGE_AMOUNT } from "@planx/components/SetFee/model";
+import { getQuarter } from "date-fns";
+import sumBy from "lodash/sumBy";
+import React from "react";
+import SettingsSection from "ui/editor/SettingsSection";
+
+import { SubscriptionProps } from "../types";
+import {
+  getUKFiscalYear,
+  getUKFiscalYearQuarter,
+  sumServiceCharges,
+} from "../utils";
+
+const DISCOUNT_THRESHOLD = 10000; // £10k
+
+export const Discount = ({ serviceCharges }: SubscriptionProps) => {
+  // "This fiscal year" is relative to when you access this page
+  const thisQuarter = getUKFiscalYearQuarter(getQuarter(new Date()));
+  const thisFiscalYear = getUKFiscalYear(thisQuarter, new Date().getFullYear());
+
+  const serviceChargesThisFiscalYear = serviceCharges.filter(
+    (sc) => sc.fiscalYear === thisFiscalYear,
+  );
+
+  return (
+    <SettingsSection background>
+      <Typography variant="h3" component="h4" gutterBottom>
+        Discount
+      </Typography>
+      <Typography variant="body2" color="text.secondary">
+        Service charges exceeding £10k per fiscal year count as a discount
+        towards your next renewal cost of Plan✕.
+      </Typography>
+      {serviceChargesThisFiscalYear.length > 0 ? (
+        <DiscountProgress serviceCharges={serviceChargesThisFiscalYear} />
+      ) : (
+        <Typography variant="body1">{`No service charges found.`}</Typography>
+      )}
+    </SettingsSection>
+  );
+};
+
+const DiscountProgress = ({ serviceCharges }: SubscriptionProps) => {
+  const rawAmountCollected = sumBy(serviceCharges, "amount");
+  const percentOfThreshold = (rawAmountCollected / DISCOUNT_THRESHOLD) * 100;
+
+  const amountToGo = DISCOUNT_THRESHOLD - rawAmountCollected;
+  const numberSubmissionsToGo = amountToGo / DEFAULT_SERVICE_CHARGE_AMOUNT;
```

Comment:
Currently, all teams will use the `DEFAULT_SERVICE_CHARGE_AMOUNT` - but in the future we may need to adjust this to account for different amounts per team.